packages/roosterjs-editor-api/lib/format/getFormatState.ts

```diff
@@ -45,6 +45,8 @@ function queryCommandState(editor: Editor, command: string): boolean {
 /**
  * Get format state at cursor
  * @param editor The editor
+ * @param event (Optional) The plugin event
```

Comment:
Explain why there is this parameter. What's the difference when pass or not pass this parameter.